swap/swap_test.go

```diff
@@ -223,40 +213,22 @@ func TestStoreBalances(t *testing.T) {
 	if err != nil {
 		t.Error("Unexpected balance update failure.")
 	}
-	// store balance peers should now include both test peers
-	expectedBalancePeers = []string{balanceKey(testPeerID), balanceKey(testPeer2ID)}
-	compareBalancePeers(t, s, expectedBalancePeers)
 	// store balance for each peer should match
 	comparePeerBalance(t, s, testPeerID, peerBalance)
 	comparePeerBalance(t, s, testPeer2ID, peer2Balance)
 }
 
 func comparePeerBalance(t *testing.T, s *Swap, peer enode.ID, expectedPeerBalance int64) {
 	var peerBalance int64
-	err := s.stateStore.Get(balanceKey(peer), &peerBalance)
-	if err != nil {
+	err := s.store.Get(balanceKey(peer), &peerBalance)
```

Comment:
dont we want `t.Helper()` here?

Reply:
You're right. Fixed.

Reply:
Also added the call to functions `testBalances`, `testPeerBookings` and `verifyBookings`.